### PR TITLE
edge-lb: add mesos-listener metrics [1.13.x]

### PR DESCRIPTION
**Pull request by vespian**
Jira: https://jira.mesosphere.com/browse/DCOS-54024
`master` PR: https://github.com/dcos/grafana-dashboards/pull/80

### CODE REVIEW

**Comment by vespian** [comment]
@philipnrmn @gracedo @branden Please have a look

**Comment by vespian** [comment]
@philipnrmn @gracedo @branden Please have a look

**Comment by vespian** [comment]
The cluster will be up until ~20:40 PST.

http://54.213.235.148/service/dcos-monitoring/grafana/d/xUjCXjDWz/edge-lb-apiserver-mesos-listener?panelId=7&orgId=1&var-dcos_cluster_name=All&var-applicationPath=All&from=now-30m&to=now&tab=general

File: dashboards/Edge-LB/Edge-LB-Apiserver_mesos-listener.json [hunk 1]
```diff
@@ -0,0 +1,625 @@
...
+      "6h",
+      "12h",
+      "24h",
+      "2d",
+      "7d",
+      "30d"
+    ]
+  },
+  "timezone": "",
+  "title": "Edge-LB: Apiserver - mesos-listener",
```
Comment (gracedo) [inline dashboards/Edge-LB/Edge-LB-Apiserver_mesos-listener.json]: Can we use `Mesos Listener` instead of `mesos-listener`?

**Comment by vespian** [comment]
@gracedo Ready for another pass

**Review by gracedo — APPROVED** [review]

**Pull request merged**